README.md

```diff
@@ -41,51 +52,54 @@ properties:
   once? Defaults to `false`.
```

Comment:
We should flip this to `true`. It is more likely that a warning should be issued an infinite number of times than a finite number. Singular warnings will get lost in regular application logs.

Reply:
Logs cost money often, so I like that the explicit `unlimited` bool so devs know that the warning is being emitted more and more times

Reply:
That is true. And I will concede that most users of this library are probably writing their own libraries for which they want users to be aware of deprecations, and they likely won't consider that detail. But I think that is also an argument for making the default `true`: it is the most common intention. People the warnings are meant to educate are unlikely to see a one off warning, and will then complain when the feature they were relying has been removed.

@fastify/core please cast a vote here:
👍 => `true`
👎 => `false`